gaurun/client.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/mercari/gcm"
+	"github.com/mercari/gaurun/service/google"
```

Comment:
I'm not sure about terraform case. What I mean that there is not strong reason that changing original package name. **gcm** is still better.